compose.override.example.yaml

```diff
@@ -18,16 +18,13 @@ services:
       - "${COMPOSER_CACHE_DIR:-${HOME}/.cache/composer}:/home/php/.composer/cache:z"
       - "${COMPOSER_HOME:-${HOME}/.composer}/auth.json:/home/php/.composer/auth.json:z"
       - "${COMPOSER_HOME:-${HOME}/.composer}/config.json:/home/php/.composer/config.json:z"
-      - ".docker-volume/starship.toml:/home/php/.config/starship.toml:rw,z" # allow for local edits to the integrated shell
-      - ".docker-volumes/atuin:/home/php/.local/share/atuin/:rw,z" # allow to keep the shell history stored in Atuin between container restarts
+      # allow for local edits to the integrated Starship shell customization
+      # Note: before adding this line make sure that the file exists (even empty), otherwise Docker will mount as a folder
+      - ".docker-volume/starship.toml:/home/php/.config/starship.toml:rw,z"
+      # allow to keep the shell history stored in Atuin between container restarts
```

Comment:
does the folder on the host also need to exist for this to actually work, or is it created automatically?

Reply:
it will be automatically created. the issue with the first one is that we are mounting a "FILE", and Docker by default (if the source of the volume does not exist) will mount a folder (named `starship.toml`), not a file